src/types/types.js

```diff
@@ -6,7 +8,7 @@ export type Layer = {
   y1: number,
   x2: number,
   y2: number,
-  options?: { [string]: string }
+  options?: TODO
 };
 
 export type LayerPredicate = (layer: Layer) => boolean;
```

Comment:
Not at al vital, but if you want you could make this a generic predicate type so we can reuse it for other types.

So instead of this:

```js
type LayerPredicate = (layer: Layer) => boolean;
const foo: LayerPredicate = layer => layer.id === id;
```

We have this:

```js
type Predicate<T> = (item: T) => boolean;
const foo: Predicate<Layer> = layer => layer.id === id;
const bar: Predicate<SketchbookDocument> = doc => doc.layers.length > 0;
```

We don't actually have any other cases where we're using a predicate like this yet, so you could argue it's premature abstraction to generalise it at this stage.